cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -4055,14 +4087,15 @@ search = | tstats `cs_summariesonly_authentication` count, min(_time) as firstTi
     [| stats sum(count) as count by app, src, user \
     | eval new_user = user." (".count.")" \
     | fields - count] \
-| stats sum(count) as count, min(firstTime) as firstTime, max(lastTime) as lastTime, values(signature) as signature, values(new_*) as * by app, src \
+| stats sum(count) as count, min(firstTime) as firstTime, max(lastTime) as lastTime, values(signature) as signature, values(is_privileged_user) as is_privileged_user, values(new_*) as * by app, src \
 | where count > `cs_authentication_bruteforce_attempt_limit` \
 | `cs_human_readable_time_format(firstTime)` \
 | `cs_human_readable_time_format(lastTime)` \
 | eval reasons=mvjoin(signature, ", ") | fields - signature | eval users=mvjoin(user, ", ") | fields - user \
-| eval cyences_severity = case(count>1000, "critical", count>500, "high", count>100, "medium") \
+| eval cyences_severity = if(is_privileged_user=="Yes", case(count>500, "critical", count>100, "high"), case(count>1000, "critical", count>500, "high", count>100, "medium")) \
```

Comment:
This is considered even if only one of the user is previledged right?

Reply:
Yes, correct. here, is_privileged_user field will be multivalued but search if one if the user is privileged from that.

Reply:
lg